plugin/src/main/java/org/opensearch/ml/action/models/UpdateModelTransportAction.java

```diff
@@ -593,17 +613,27 @@ private UpdateRequest createUpdateModelGroupRequest(
     ) {
         modelGroupSourceMap.put(MLModelGroup.LATEST_VERSION_FIELD, updatedVersion);
         modelGroupSourceMap.put(MLModelGroup.LAST_UPDATED_TIME_FIELD, Instant.now().toEpochMilli());
-        UpdateRequest updateModelGroupRequest = new UpdateRequest();
-
+        /* Old code here. TODO investigate if we need to add seqNo and primaryTerm to data object request
+        UpdateRequest updateModelGroupRequest = new UpdateRequest();        
```

Comment:
I think we added this for VersionConflictEngineException. This occurs when maybe there are two updates happening at once. So I assume you are planning to have a discussion on this tomorrow, right?

Reply:
Yes, we need some version of this and will address this in another PR.